packages/space-opera/src/components/model_viewer_snippet/components/download_button.ts

```diff
@@ -120,8 +143,40 @@ async function prepareZipArchive(
     zip.file(relativeFilePaths.iosName!, response.blob());
   }
 
+  let template = modelViewerTemplate;
+  // Conditionally set script if anything requires javascript. Currently, this
+  // is only the progressbar.
+  const script =
+      bestPractices.progressBar ? '<script src="script.js"></script>' : '';
+  const snippet = beautify_snippet(data.snippetText.split('\n'));
+
+  // Replace placeholders in html strings with content
+  template = template.replace('<div>modelviewer</div>', snippet);
+  template = template.replace('<script>script</script>', script);
```

Comment:
Okay, I guess you are in fact doing what I asked above. Might be clearer to simply add this instead of replacing a dummy.